spec/didparameters.md

```diff
@@ -4,7 +4,24 @@ This section describes the support of the `did:webs` method for certain DID para
 
 ### Support for `versionId`
 
-TODO, see https://github.com/trustoverip/tswg-did-method-webs-specification/issues/24
+The `did:webs` DID method supports the `versionId` DID parameter. This DID parameter is defined
+[here](https://www.w3.org/TR/did-core/#did-parameters). (TODO: Add proper references to external documents).
+
+This allows clients to instruct a DID Resolver to return a specific version of a DID document, as opposed to the
+latest version. The `did:webs` DID method is ideally suited for this functionality, since a continuous, self-certifying
+stream of events lies at the heart of the DID method's design, see section [KERI Fundamentals](#keri-fundamentals).
+
+Valid values for this DID parameter are sequence numbers of events in the [[ref: KERI event stream]]. When a `did:webs`
```

Comment:
This is very interesting.  When the controller of a DID puts a reference to the DID into a document (say, an issuer put the DID into a verifiable credential), they can use the `?versionId=<N>` form of the DID where `<N>` is the current Version Id (KERI sequence number).  Anyone receiving the DID reference can then know what version of the DIDDoc to have resolved.  That’s very useful!

Reply:
Yes. In addition, the `versionId` is also returned in the DID document metadata, see https://github.com/trustoverip/tswg-did-method-webs-specification/pull/49. And since this is defined in DID Core, this works consistently across different DID methods (but of course other DID methods can't "guarantee" this with the same reliability as did:keri/did:webs).